submissions/LGeorge007/memory-pair-game/script.js

```diff
@@ -0,0 +1,65 @@
+const allCards = document.querySelectorAll(".flipper");
+const arrOfCards = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19];
+let unopenedCards = arrOfCards.length;
+let firstCard;
+let secondCard;
+let clikCounter = 0;
+
+const checkCard = function (event) {
+    const checkMatch = function () {
+        if (clikCounter == 0) {
+            firstCard = event.path[1];
+            clikCounter++;
+        } else if (clikCounter == 1) {
+            secondCard = event.path[1];
+            //The same card must not be removed
+            if (firstCard.style.order != secondCard.style.order) {
```

Comment:
Scripting based on CSS rules is bad in general. If you wan't to tie your html cards with javascript it is better to use `ID`s.

Reply:
How can I organize cards using ID? 
Using flexbox, I may apply order property to organize cards.

Reply:
That is why your way of dealing with cards has a lot of flaws. You already learned methods for modifying the html structure using JS code. Wouldn't it be better to create a template in JavaScript and create cards layout using JavaScript array methods?

Reply:
Fixed. I use ID="flipper" isntead .flipper.